test/Interop/Cxx/value-witness-table/custom-destructors-non-virtual.swift

```diff
@@ -0,0 +1,125 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+//
+// REQUIRES: executable_test
+
+import CustomDestructor
+import StdlibUnittest
+
+var CXXDestructorTestSuite = TestSuite("CXXDestructor")
+
+protocol InitWithPtr {
+  init(value: UnsafeMutablePointer<Int32>!)
+}
+
+extension HasUserProvidedDestructor : InitWithPtr { }
+
+protocol InitWithMember {
+  init(member: HasUserProvidedDestructor)
+}
+
+extension HasEmptyDestructorAndMemberWithUserDefinedConstructor
+  : InitWithMember { }
+
+// These are all maked as "@inline(never)" so that the destructor is invoked in
+// this function and not in the caller (after being inlined).
```

Comment:
Remove this comment.

This test should probably also be run with optimizations.